wp-content/themes/eighshift-task/src/blocks/layout/footer/_footer-navigation.scss

```diff
@@ -0,0 +1,31 @@
+$footer-navigation: (
+  horizontal-spacing: $base-gutter-size,
+  text: $base-nobel-color,
```

Comment:
what text? if you read this key name in the code will you know what is assigned to that variable name?